source/common/stats/histogram_impl.cc

```diff
@@ -22,7 +31,13 @@ const std::vector<double>& HistogramStatisticsImpl::supportedQuantiles() const {
   return supported_quantiles;
 }
 
-std::string HistogramStatisticsImpl::summary() const {
+const std::vector<double>& HistogramStatisticsImpl::supportedBuckets() const {
+  static const std::vector<double> supported_buckets = {0.005, 0.01, 0.025, 0.05, 0.1, 0.25,
```

Comment:
just curious, are these standard bucket sizes used/exported by prometheus generally?

Reply:
In a subsequent PR I want to make this configurable via bootstrap config. I am open to suggestions if we want to expand this initial list though for the time being. 

I wanted to capture a range of normal request timings, we could possibly add a few more buckets tending towards higher request timings (eg: 1 hour) to capture this data.